### PR TITLE
Added common linters

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Any contribution that you make to this repository will
+be under the BSD license 2.0, as dictated by that
+[license](https://opensource.org/licenses/BSD-3-Clause).
```
Comment (clalancette) [inline CONTRIBUTING.md]: This is actually incorrect (there is no such thing as the BSD license 2.0; I don't know what I was thinking when I added that to ament_lint all those years ago).  This should be:

```suggestion
Any contribution that you make to this repository will
be under the 3-Clause BSD License, as dictated by that
[license](https://opensource.org/licenses/BSD-3-Clause).
```

File: src/laser_geometry/__init__.py [hunk 2]
```diff
@@ -1 +1,29 @@
-from .laser_geometry import LaserProjection
+# Copyright (c) 2024, Open Source Robotics Foundation, Inc.
+# All rights reserved.
+# Redistribution and use in source and binary forms, with or without
```
Comment (clalancette) [inline src/laser_geometry/__init__.py]: This should probably be:

```suggestion
# Copyright (c) 2014, Enrique Fernandez
# All rights reserved.

# Redistribution and use in source and binary forms, with or without
```

(it came in with commit d4785e879a06f2dc7c66cae58c742021677e0966 that added `laser_geometry.py` below)

File: src/laser_geometry/laser_geometry.py [hunk 3]
```diff
@@ -1,42 +1,41 @@
...
-CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
-SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
-INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
-CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
-ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
-POSSIBILITY OF SUCH DAMAGE.
-"""
+# Copyright (c) 2014, Enrique Fernandez
+# All rights reserved.
+# Redistribution and use in source and binary forms, with or without
```
Comment (clalancette) [inline src/laser_geometry/laser_geometry.py:4]:
```suggestion

# Redistribution and use in source and binary forms, with or without
```

File: test/projection_test.py [hunk 4]
```diff
@@ -1,23 +1,51 @@
 #!/usr/bin/env python3
```
Comment (clalancette) [inline test/projection_test.py]:
```suggestion
```

File: test/projection_test.py [hunk 5]
```diff
@@ -1,23 +1,51 @@
...
+# Copyright (c) 2024, Open Source Robotics Foundation, Inc.
+# All rights reserved.
+# Redistribution and use in source and binary forms, with or without
```
Comment (clalancette) [inline test/projection_test.py]: This should likely be:

```suggestion
# Copyright (c) 2014, Enrique Fernandez
# All rights reserved.

# Redistribution and use in source and binary forms, with or without
```

(it came in with 0ed1dafb145039ff58cc6b141893713dc918d867, around the same time as the rest)

**Review by clalancette — CHANGES_REQUESTED** [review]

**Comment by ahcorde** [comment]
friendly ping @clalancette

File: package.xml [hunk 6]
```diff
@@ -16,14 +16,14 @@
 
   <license>BSD</license>
 
+  <url type="website">http://ros.org/wiki/laser_geometry</url>
```
Comment (clalancette) [inline package.xml]: Sorry, I missed this earlier.  We shouldn't point to the wiki anymore, but instead:

```suggestion
  <url type="website">http://github.com/ros-perception/laser_geometry</url>
```

File: src/laser_geometry/laser_geometry.py
Comment (clalancette) [inline src/laser_geometry/laser_geometry.py:1]: We shouldn't give the executable bit to this file.  That is, we should leave the permissions as 100644.

**Review by clalancette — CHANGES_REQUESTED** [review]
Two more small changes, then I think this will be good to go.

File: src/laser_geometry/laser_geometry.py
Comment (ahcorde) [reply src/laser_geometry/laser_geometry.py:1]: https://github.com/ros-perception/laser_geometry/pull/96/commits/c209925257d55c1f744cde70b185ea163c5d6fea

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21668/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/15952/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/1171/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22383/)

**Review by clalancette — APPROVED** [review]

**Pull request merged**